corehq/apps/case_search/const.py

```diff
@@ -45,67 +45,66 @@ def es_field_name(self):
         return self._es_field_name or self.system_name
 
 
-# System properties are top level, schema'd properties on CommCareCase that are
-# made available to the user in interactions with the case search elasticsearch
+# These are top level, schema'd properties on CommCareCase that are made
+# available to the user in interactions with the case search Elasticsearch
 # index.
 # These are stored along with dynamic case properties in the case search index
 # to be easily searchable, then removed when pulling the case source from ES.
-SPECIAL_CASE_PROPERTIES_MAP = {prop.key: prop for prop in [
-    _SystemProperty(
+INDEXED_METADATA_BY_KEY = {prop.key: prop for prop in [
```

Comment:
What I'm hoping to convey here is that these are metadata case properties (ID, type, status, dates, etc) that are indexed in a way to be searchable by users.

Reply:
I do worry that `INDEXED_METADATA` makes it harder to understand on first glance, but can see how it is a richer name. I do like `INDEXED_CASE_PROPERTIES` since that seems like enough to differentiate from dynamic case properties, but also don't have much context here/strong feelings.